content/intro-to-storybook/ember/en/screen.md

```diff
@@ -8,53 +8,66 @@ We've concentrated on building UIs from the bottom up; starting small and adding
 
 In this chapter we continue to increase the sophistication by combining components in a screen and developing that screen in Storybook.
 
-## Nested container components
+## Screen components
 
-As our app is very simple, the screen we’ll build is pretty trivial, simply wrapping the `TaskList` component (which supplies its own data via Redux) in some layout and pulling a top-level `error` field out of redux (let's assume we'll set that field if we have some problem connecting to our server).
+As our app is very simple, the screen we’ll build is pretty trivial, simply
+wrapping the `<TaskList>` component in some layout and apply a data layer with
+loading and error states from redux (let's assume we'll set that field if we have some
+problem connecting to our server). We will do that for two reasons: First is to
+keep data-management and presentation clearly separated and second screen
+components do help to move things around in the app as well as design reviews
+within Storybook.
 
-Let's start by updating the store (in `app/reducers/index.js`) to include the error field we want:
+Let's start by updating reducers and actions to include the
```

Comment:
@gossi the reducer should be singular if you don't mind. This should be: "Let's start by updating the reducer and the actions to include....."